core/src/test/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationInfoSuite.scala

```diff
@@ -895,4 +895,24 @@ class ApplicationInfoSuite extends FunSuite with Logging {
       }
     }
   }
+
+  test("test cpu reused subquery") {
```

Comment:
There is an eventlog `nds_q86_test` that contains `ReusedExchange`.
The issue with using an eventlog as an input is that we get stuck with the Spark-version that was used to generate that eventlog. In some situations, we found that different spark-versions would be different and we do not catch that difference early on.
I will check if I can figure out something to generate eventlog on the fly for a `ReusedSubquery`